.genignore

```diff
@@ -1 +1,3 @@
 LICENSE
+src/main/java/com/styra/opa/Porcelain.java
+src/main/java/com/styra/opa/PorcelainException.java
```

Comment:
```suggestion
src/main/java/com/styra/opa/OPA.java
src/main/java/com/styra/opa/OPAException.java
```
[question]: Should these be updated to the above suggestion? IIRC, Speakeasy will nuke your hard work if the names aren't updated. :sweat_smile:

Reply:
Good catch!

Reply:
Closing this as I did this a while ago